streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -766,6 +783,20 @@ private void checkIfUnexpectedUserSpecifiedConsumerConfig(final Map<String, Obje
         return props;
     }
 
+    /**
+     * Get the configs fro the {@link org.apache.kafka.clients.admin.AdminClient}.
```

Comment:
typo `fro` -> `for`

Nit: `{@link org.apache.kafka.clients.admin.AdminClient}` -> `{@link org.apache.kafka.clients.admin.AdminClient AdminClient}`